vmsyscall/src/bootinfo.rs

```diff
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: Apache-2.0
+
+//! The `bootinfo` module includes the hard-coded memory layout and
+//! the `BootInfo` structure which is handed to the kernel.
+//!
+//! Initially copied from:
+//! https://github.com/rust-osdev/bootloader/blob/90f5b8910d146d6d489b70a6341d778253663cfa/src/bootinfo/mod.rs
+
+use crate::memory_map::MemoryMap;
+use core::fmt;
+
+/// Hard-coded trigger port
+pub const SYSCALL_TRIGGER_PORT: u16 = 0xFF;
```

Comment:
oh, we can move that into `vmrun` ... doesn't have to be here anymore, because it's a BootInfo argument